backend/scripts/sweepstakes-email.ts

```diff
@@ -0,0 +1,127 @@
+import { runScript } from 'run-script'
+import fetch from 'node-fetch'
+
+const MAILGUN_API_KEY = '' // add API key
```

Comment:
we already have this in the secrets, you should access it in process.env in the runScript function body